client/src/components/Main/FeatureSection/FeatureSection.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ItemLists from '@/components/base/ItemLists/ItemLists';
+
+import { baeminFont } from '@/static/style/common';
+
+/**
+ * TODO:
+ * mode 에 따라 api 호출해서
+ * state update 할 것
+ */
+const FeatureSection = (props) => {
+  let title = '';
+
+  if (props.mode === 'new') {
+    title = '새로 나왔어요';
+  } else if (props.mode === 'best') {
+    title = '잘나가요';
+  } else if (props.mode === 'discount') {
+    title = '지금은 할인 중';
+  }
```

Comment:
```suggestion
import { FC } from 'react';

export enum Badge {
  New = 'New',
  Best = 'Best',
  Discount = 'Discount',
  Green = 'Green',
}

export const badgeTitleObj = {
  [Badge.New]: '새로 나왔어요.',
  [Badge.Best]: '잘나가요',
  [Badge.Discount]: '지금은 할인 중',
  [Badge.Green]: '친환경이에요',
}

export interface FeatureSectionProps {
  mode: Badge;
  items: Item[];
}

const FeatureSection: FC<FeatureSectionProps> = ({mode, items}) => {
  const title = titleObj[mode];
```
1. `FC<generic>`을 사용하여 리액트 컴포넌트 타입을 정할 수 있습니다. 
2. `enum`을 사용하면 편하고 활용성이 더 클 거 같습니다. 
3. Mode, titleObj, FeatureSectionProps를 사용하게 된다면 constant.ts파일로 나누어도 좋을 거 같습니다.

Reply:
FC를 쓰려다가 벨로퍼트가 FC에 대한 에러가 있다고 해서 권장을 안해서 안쓰고 있었습니다ㅠㅠ,, 그리고 enum은 아래 코멘트와 같이 조사해보고 오겠습니다!